src/site/paragraphs/table.drupal.liquid

```diff
@@ -27,31 +27,37 @@
     }
 {% endcomment %}
 
-
-<table data-template="paragraphs/table" data-entity-id="{{ entity.entityId }}" class="va-table">
-
-    {% if entity.fieldTable.caption %}
-        <caption>{{ entity.fieldTable.caption }}</caption>
-    {% endif %}
-
-    {% for value in entity.fieldTable.value %}
-        {% if forloop.first == true %}
-            <thead>
-                <tr>
-                    {% for column in value %}
-                        <th>{{ column }}</th>
-                    {% endfor %}
-                </tr>
-            </thead>
-        {% elsif forloop.last == false %}
-            <tbody>
-                <tr class="class">
+<table data-template="paragraphs/table" data-entity-id="{{ entity.entityId }}" class="va-table rwd" role="table">
+    <tbody>
+        {% if entity.fieldTable.caption %}
+            <caption>{{ entity.fieldTable.caption }}</caption>
+        {% endif %}
+        {% assign colLabels = entity.fieldTable.value.0  %}
+        {% for value in entity.fieldTable.value %}
+            {% if forloop.first == true %}
+                <thead>
+                    <tr role="row">
+                        {% for column in value %}
+                            <th role="columnheader" scope="col">{{ column }}</th>
+                        {% endfor %}
+                    </tr>
+                </thead>
+            {% elsif forloop.last == false %}
+                <tr class="vads-u-padding-top--5" role="row">
                     {% for column in value %}
-                        <td>{{ column }}</td>
+                        <dfn>
+                            <td class="vads-u-padding-bottom--0 column-label vads-u-font-weight--bold medium-screen:vads-u-display--none" role="cell">
+                                {{colLabels | getArrayItemByIndex: forloop.index0}}
```

Comment:
I haven't seen `forloop.index0` before. Are we just trying to get the first element out of `colLabels`?

Reply:
`index0` is zero based iteration whereas `index` starts off at 1... smh drupal :P
we are trying to get the associated col label (From what used to be the table header, which is hidden in mobile) for each column value

Reply:
Wow yeah, I see it in the Shopify docs now, https://shopify.dev/docs/themes/liquid/reference/objects/for-loops. Thanks for the insight!